app/services/facebook_authenticator.rb

```diff
@@ -12,10 +12,14 @@ def initialize(auth_code)
   end
 
   def authenticate!
+    access_token  = get_access_token(@auth_code)
```

Comment:
yeah, before I was doing the opposite, as well as extracting these temp vars into their own methods. remember yesterday it was hard to navigate this... though it better to be explicit?